public/tier0/dbg.h

```diff
@@ -222,7 +222,7 @@ DBG_INTERFACE void SetAssertFailedNotifyFunc( AssertFailedNotifyFunc_t func );
 DBG_INTERFACE void CallAssertFailedNotifyFunc( const char *pchFile, int nLine, const char *pchMessage );
 
 /* True if -hushasserts was passed on command line. */
-DBG_INTERFACE bool HushAsserts();
+//DBG_INTERFACE bool HushAsserts();

```

Comment:
Why comment this? Can a note be added to the file.

Reply:
I undid the comment now. It wasn't meant to be included - only had it locally as a temporary change for testing https://github.com/alliedmodders/hl2sdk/commit/d0c71a9eb7e202d42ad77b8573dc5b5d44cb5ae8